config/internal/mariadb/route.yaml.tmpl

```diff
@@ -0,0 +1,14 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: mariadb-{{.Name}}
+  namespace: {{.Namespace}}
+  labels:
+    app: mariadb-{{.Name}}
+    component: data-science-pipelines
+spec:
+  to:
+    kind: Service
+    name: mariadb-{{.Name}}
+  port:
+    targetPort: 3306
```

Comment:
we probably didn't think this one through @gmfrasca since sql does not support http/https is which is what ocp routes communicate over, sql uses `tds` so we can't use a route to connect to it. @hbelmiro to unblock you here I think we should probably remove database support for routes for now, as that's not possible.

Reply:
Route removed. Can you please have a second look @HumairAK?

Reply:
@HumairAK @gmfrasca I also edited the [issue description](https://github.com/opendatahub-io/data-science-pipelines-operator/issues/220#issue-1819015326).